recipes-bsp/flash-script/flash-script/imx8mp-var-dart-hmx1/flash.cmd

```diff
@@ -0,0 +1,55 @@
+env set usbdev 0
+env set usbpart 1
+env set wic_file hmx-image.wic.gz
+env set set_red_led "i2c mw 0x30 7.1 0x50 1;i2c mw 0x30 4.1 0x4 1"
+
+#set white led flashing faster.
+env set white_blink_fast "i2c mw 0x30 1.1 1 1;"
+run set_white_flashing_led; 
+run white_blink_fast;
+
+# TODO set watchdog (wdt start) 600000 milliseconds = 10 minutes timeout for programming
+
+# Start USB device and try to load wic_file into memory if load fails, exit the script (and boot normally).
+if test -n "$usb_flash_wanted"; then 
+    env set loadimage_usb "load usb ${usbdev} ${img_addr} ${wic_file}"
+    echo Loading ${wic_file} from USB ${usbdev};
```

Comment:
Will not make this chage. this will be thre result.
if test -n "$usb_flash_wanted"; then echo hej; fi

if test -n $usb_flash_wanted; then echo hej; fi
hej